test/unit/generator/validate-filename.test.js

```diff
@@ -0,0 +1,45 @@
+const moment = require('moment')
+const getFilename = require('../../../app/generator/create-filename')
+const { STATEMENT } = require('../../../app/constants/document-types')
+
+let timestamp
+let errorStatement
```

Comment:
use mockStatement

Reply:
removed errorStatement and switched to mockStatement